src/main/java/com/google/sps/Trip.java

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+
+/**
+ * Trip is the class for storing a single trip (could be multiple days).
+ */
+public class Trip {
+  private String tripName;
+  private String startDate;
+  private String endDate;
+  private int numDays;
+  private ArrayList<TripDay> tripDays;
```

Comment:
No action required, just a note. Eventually, I think we'll add a `String tripId` variable that holds a 16-digit alphanumeric identifier, so that we can easily store and access different trips in datastore. Again, no action required on this, just a note for later plans.